routes/drinks.js

```diff
@@ -0,0 +1,276 @@
+import express from 'express';
+import { createCloudinaryFolder, setupImageFolders, uploadImageToFolder } from '../db_config/cloudinary_config.js';
+import pool from '../db_config/db_connection.js'
+import { config } from 'dotenv';
+config();
+
+import swaggerUi from 'swagger-ui-express';
+import YAML from 'yamljs';
+
+const router = express.Router();
+
+const swaggerDocument = YAML.load('./documentary/swagger-specs.yaml');
+
+// mounting the Swagger UI middleware:
+router.use('/api-docs', swaggerUi.serve);
```

Comment:
You mounted this in the app.js file and you are mounting here again?

Reply:
adjusted now